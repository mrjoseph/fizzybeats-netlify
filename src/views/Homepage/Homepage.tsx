import React from 'react';
import Container from '@material-ui/core/Container';
import SocialBar from '../../components/SocialBar';
import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
const useStyles = makeStyles((theme: Theme) =>
  createStyles({
    container: {
      margin: 'auto',
      width: '1024px',
      bckground: 'red'
    },
  }),
);

const Homepage = () : JSX.Element | null => {
  const classes = useStyles();
    return (
    <React.Fragment>
        <Container className={classes.container}>
          <SocialBar />
      </Container>
    </React.Fragment>
    )
}

export default Homepage;