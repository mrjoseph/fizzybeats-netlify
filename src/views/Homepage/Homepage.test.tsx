  
import React from 'react';
import Homepage from './Homepage';
import renderer from 'react-test-renderer';
describe('SocialBar', () => {
    describe('snapshot', () => {
        it('renders correctly', () => {
            interface Ihomepage {}    
            const props:Ihomepage = {}
            const tree = renderer.create(<Homepage {...props}/>).toJSON();
            expect(tree).toMatchSnapshot();
          });
    })
});