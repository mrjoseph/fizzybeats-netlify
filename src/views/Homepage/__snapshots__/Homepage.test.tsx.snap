// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialBar snapshot renders correctly 1`] = `
<div
  className="MuiContainer-root MuiContainer-maxWidthXl"
>
  <div
    className="MuiTypography-root MuiTypography-body1"
    style={
      Object {
        "backgroundColor": "#cfe8fc",
        "height": "100vh",
      }
    }
  >
    social media linkss
  </div>
</div>
`;
